yarn.lock

```diff
@@ -10126,16 +10126,11 @@ flat-cache@^3.0.4:
     flatted "^3.1.0"
     rimraf "^3.0.2"
 
-flatpickr@^4.6.3:
+flatpickr@4.6.9, flatpickr@^4.6.3, flatpickr@~4.5.2:
```

Comment:
I don't see any changes in any `package.json`, how come the yarn.lock is different?

Reply:
Rebase on `master` will fix it.